lib/middleware/bearerToken.ts

```diff
@@ -41,7 +41,7 @@ const validate = (
  * @throws
  * This exception is thrown if the bearer token is not found
  */
-const getBearerToken = (req: NextApiRequest) => {
+export const getBearerToken = (req: NextApiRequest): string => {
```

Comment:
This could probably be better names from getBearerToken to `extractTokenFromAuth` or something along those lines to make it clear what exactly it is doing.

Reply:
Renamed  as `extractBearerTokenFromReq`